doc/enterprise/audit_log.rst

```diff
@@ -45,115 +258,215 @@ decide whether you need to take actions:
+        *   -   Space dropped
+            -   ``space_drop``
+            -   ``INFO``
             -   ``Drop space <SPACE>``
-        *   -   Tuple inserted into space     
-            -   ``space_insert``  
+        *   -   Tuple inserted into space
+            -   ``space_insert``
+            -   ``VERBOSE``
             -   ``Insert tuple <TUPLE> into space <SPACE>``
-        *   -   Tuple replaced in space   
-            -   ``space_replace``   
+        *   -   Tuple replaced in space
+            -   ``space_replace``
+            -   ``VERBOSE``
             -   ``Replace tuple <TUPLE> with <NEW_TUPLE> in space <SPACE>``
-        *   -   Tuple deleted from space   
-            -   ``space_delete``   
+        *   -   Tuple deleted from space
+            -   ``space_delete``
+            -   ``VERBOSE``
             -   ``Delete tuple <TUPLE> from space <SPACE>``
 
 
     ..  note::
-    
+
         The ``eval`` event displays data from the ``console`` module
         and the ``eval`` function of the ``net.box`` module.
         For more on how they work, see :ref:`Module console <console-module>`
-        and :ref:`Module net.box -- eval <net_box-eval>`. 
+        and :ref:`Module net.box -- eval <net_box-eval>`.
         To separate the data, specify ``console`` or ``binary`` in the session field.
 
-.. _audit-log-event-groups:
+..  _audit-log-structure:
+
+Structure of audit log event
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each audit log event contains a number of fields that can be used to filter and aggregate the resulting logs.
+An example of a Tarantool audit log entry in JSON:
+
+..  code-block:: json
+
+    {
+        "time": "2024-01-15T13:39:36.046+0300",
+        "uuid": "cb44fb2b-5c1f-4c4b-8f93-1dd02a76cec0",
+        "severity": "VERBOSE",
+        "remote": "unix/:(socket)",
+        "session_type": "console",
+        "module": "tarantool",
+        "user": "admin",
+        "type": "auth_ok",
+        "tag": "",
+        "description": "Authenticate user Admin"
+    }
+
+Each event consists of the following fields:
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 30 35 35
+        :header-rows: 1
+
+        *   -   Field
+            -   Description
+            -   Example of a log field display
+        *   -   ``time``
+            -   Time of the event
+            -   ``2024-01-15T16:33:12.368+0300``
+        *   -   ``uuid``
+            -    Since :doc:`3.0.0 </release/3.0.0>`. A unique identifier of audit log event
+            -   ``cb44fb2b-5c1f-4c4b-8f93-1dd02a76cec0``
+        *   -   ``severity``
+            -   Since :doc:`3.0.0 </release/3.0.0>`. A severity level. Each system audit event has a severity level determined by its importance.
+                :ref:`Custom events <audit-log-custom>` have the ``INFO`` severity level by default.
+
+            -   ``VERBOSE``
+        *   -   ``remote``
+            -   Remote host that triggered the event
+            -   ``unix/:(socket)``
```

Comment:
An IP address example might be more real-life but let's keep it as is. Just a random thought :)